src/resources/js/dialog.js

```diff
@@ -1,17 +1,17 @@
 var tribe = tribe || {};
 tribe.dialogs = tribe.dialogs || {};
-tribe.dialogs.dialogs = tribe.dialogs.dialogs || {};
+tribe.dialogs.dialogs = tribe.dialogs.dialogs || [];
 tribe.dialogs.events = tribe.dialogs.events || {};
```

Comment:
Should this be an array too?

Reply:
We don't use it as array, like we do with `tribe.dialogs.dialogs`: https://github.com/moderntribe/tribe-common/blob/97c251c6a8f1d72ab9545a339985de9032c57bbb/src/resources/js/dialog.js#L35